components/shared/IndexIssueListItem.js

```diff
@@ -1,14 +1,15 @@
 import React from "react";
-import Row from "react-bootstrap/Row";
-import Col from "react-bootstrap/Col";
 import Card from "react-bootstrap/Card";
 import Link from "next/link";
 import ReactMarkdown from "react-markdown";
 
 function parseBody(body) {
-  // extract content up to the first linebreak to use as description
-  const reDescription = /.*/;
-  const description = body.match(reDescription)[0];
+  // extract content up to the first sentence or linebreak if no period to use as description
+  const reDescriptionSentence = /^(.*?)[.?!]\s+/;
+  const reDescriptionLinebreak = /.*/;
+  const description =
+    body.match(reDescriptionSentence)?.[0] ||
+    body.match(reDescriptionLinebreak)[0];
```

Comment:
use the first sentence as a description unless there is no period in which use first linebreak